plugins/com.google.cloud.tools.eclipse.login.test/src/com/google/cloud/tools/eclipse/login/CredentialHelperTest.java

```diff
@@ -43,7 +43,8 @@ public void testToJsonFile() throws IOException {
     Path jsonFile = tempFolder.getRoot().toPath().resolve("credential-for-gcloud.json");
     CredentialHelper.toJsonFile(credential, jsonFile);
 
-    try (InputStream in = new FileInputStream(jsonFile.toFile());
+    try (InputStream in = Files.newInputStream(jsonFile);
+        // todo needs character set
```

Comment:
This is a design flaw in the GSON API. It should accept an InputStream, not a Reader, and autodetect the specific Unicode encoding. We can work around this, but it would pull this PR way off on a tangent. For now since this is a test, I'd prefer to leave the behavior intact.